src/ucp/api/device/ucp_host.h

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2025. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCP_HOST_H_
+#define UCP_HOST_H_
+
+#include <sys/types.h>
+#include <ucs/sys/compiler_def.h>
+#include <ucs/type/status.h>
+#include <ucp/api/ucp_def.h>
+
+
+BEGIN_C_DECLS
+
+/**
+ * @defgroup UCP_DEVICE_API Unified Communication Protocol (UCP) Device API
+ * @{
+ * This section describes UCP Device API.
+ * @}
```

Comment:
```suggestion
 * @defgroup UCP_DEVICE Device API
 * @ingroup UCP_API
 * @{
 * UCP Device API.
 * @}
```
to align with other group definitions

Reply:
fixed